New version: MeshIntegrals v0.17.0

JuliaRegistrator opened the pull request:
- Registering package: MeshIntegrals
- Repository: https://github.com/JuliaGeometry/MeshIntegrals.jl
- Created by: @mikeingold
- Version: v0.17.0
- Commit: 23d7d25e992250e9a0435b2f7a057a211b8d1090
- Reviewed by: @mikeingold
- Reference: https://github.com/JuliaGeometry/MeshIntegrals.jl/commit/23d7d25e992250e9a0435b2f7a057a211b8d1090#commitcomment-164345528
- Description: Numerical integration over Meshes.jl geometry domains
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
- This new version implements breaking changes. For the changelog and release notes see https://github.com/JuliaGeometry/MeshIntegrals.jl/releases.
`````
<!-- END RELEASE NOTES -->

<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd645a35cd7c312d03bd481ad0e60cac4fdf6456c75342b7c5e29d1e124c6d27a6e4d9c5a641cc237a6d40a46477ceaf71d08aa00ae0a2ab48ed9205ba82e4e12ca20e441835c6b1925cfc7ebf1392d5facfc45ef42000cf8440d673ba427e2ab649abbd103d460b400f081203cba444ba476d16d7ecf4acb7cb2ab4bb07e1b1d4a6f765928c6d443d599a461304af012a8f13467c584e089c7c1f50062777a0ad8fddaa2e16db95b91cdcaa1c15eb86787 -->

mikeingold commented:
Do not merge. We’ve decided to take advantage of the auto-merge blockage to incorporate another change into this release, and we will push this update to an upcoming commit.

JuliaRegistrator committed:
New version: MeshIntegrals v0.17.0

UUID: dadec2fd-bbe0-4da4-9dbe-476c782c8e47
Repo: https://github.com/JuliaGeometry/MeshIntegrals.jl.git
Tree: d62c38c7faffc0178b9f2f5d87744d5ebe459289

Registrator tree SHA: 50f504d641745716a5b3eabaf681d3a4937d2ae3
